packages/ds-medicare-gov/src/styles/components/_Forms.scss

```diff
@@ -1,23 +1,9 @@
 @use '../layout' as *;
 
 .ds-c-field {
-  border-color: var(--text-input__border-color);
-
-  color: var(--text-input__color);
-  font-family: var(--font-family-rubik);
-  font-size: var(--font-size-md);
-  padding: $spacer-2;
-  &:disabled {
-    border: solid 1px var(--text-input__border-color--disabled);
```

Comment:
The disabled border-width token in `medicare.ts`, line 430 comes from this section.